src/App.tsx

```diff
@@ -1,24 +1,361 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import * as todoService from './api/todos';
+import { TodoContext } from './todoContext';
+import { TodoList } from './todoList';
+import { TodoItem } from './todoItem';
 
-const USER_ID = 0;
+const USER_ID = 22;
+
+enum ErrorMessage {
+  'add',
+  'update',
+  'delete',
+  'title',
+}
+
+enum FilterOption {
+  'Active',
+  'Completed',
+  'All',
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [titleValue, setTitleValue] = useState('');
+  const [count, setCount] = useState(0);
```

Comment:
looks like a redundant state, it can be a simple var that will filter the array and save the array of active elements length

Reply:
idk might be but this way I have countrol over count and my checkbox logic relies on it, so if I take it out everything brakes